File: AssignmentMVVM/Model/ArtworkImage.cs
namespace AssignmentMVVM.Model
{
    public class ArtworkImage
    {
        private string imagePath;
        private Artwork artwork;

        public ArtworkImage() { }

        public ArtworkImage(string imagePath, Artwork artwork)
        {
            this.imagePath = imagePath;
            this.artwork = artwork;
        }

        public string ImagePath
        {
            get { return imagePath; }
            set { imagePath = value; }
        }

       
    }
}

File: AssignmentMVP/src/main/java/Model/ArtworkImage.java
package Model;

public class ArtworkImage
{

    private String imagePath;
    private Artwork artwork;

    public ArtworkImage() {}

    public ArtworkImage(String imagePath, Artwork artwork) {
        this.imagePath = imagePath;
        this.artwork = artwork;
    }

    public String getImagePath() { return imagePath; }
    public void setImagePath(String imagePath) { this.imagePath = imagePath; }

    public Artwork getArtwork() { return artwork; }
    public void setArtwork(Artwork artwork) { this.artwork = artwork; }
}

